src/puppetlabs/puppetdb/command.clj

```diff
@@ -427,7 +441,8 @@
         "replace facts" (replace-facts command start db blacklist-config)
         "store report" (store-report command start db)
         "deactivate node" (deactivate-node command start db)
-        "configure expiration" (configure-expiration command start db)))))
+        "configure expiration" (configure-expiration command start db)
+        "replace catalog-inputs" (replace-catalog-inputs command start db)))))
```

Comment:
I'm not sure, but wondering if we might want "replace catalog inputs" here, i.e. without the dash.  Don't think we have any precedents, but I'd tended to think of the wire command names as the plain text version, which we convert in various contexts by either `s/ /_/` or `s/ /-/`, depending.  I suppose that also has the advantage of making the conversion trivially reversible, but not sure that matters.